.github/workflows/soaking.yml

```diff
@@ -59,6 +59,14 @@ jobs:
             terraform_directory: integration-tests/dotnet/aws-sdk/wrapper
             expected_trace_template: adot/utils/expected-templates/dotnet-awssdk-wrapper.json
             soak_config: '-c 200 -m 90'
+          - name: go-awssdk-wrapper
+            layer_kind: collector
```

Comment:
Will this `layer_kind` conflict with the .NET `layer_kind` which is also `collector`? I'm especially worried if .NET & Go publish some layer called `my-collector-layer:1` and `my-collector-layer:2` then how do we know which one to use?

I think we might need to apply the same strategy of only testing Go for now as we did in `release.yml` and comment on it.

Reply:
Yeah good call. I think we should also follow the same strategy in soak test as well.

Reply:
+1 to only soak testing with Go for now.